README.md

```diff
@@ -57,30 +58,55 @@ with the command:
 ./bin/openfga run
 ```
 
-### Source
+### Building from Source
+
+There are two recommended options for building from OpenFGA source code:
+
+#### a. Building from source with `go install`
```

Comment:
In the end we show how to run the binary. Should this be "Running from source with `go install`"?

Reply:
🤷

@jon-whit @rhamzeh @miparnisari do you have any thoughts on this?

Reply:
I'd kinda prefer if we only had one recommended way. I'd vote for `go build`. It's a bit less portable as it doesn't rely on Go details as GOBIN etc.

Reply:
Maybe the conversation about preferred methods or primary methods or wording about those methods needs to be a separate discussion/RFC. For now, I think we need to get working instructions into the README.md.